sopel/plugins/handlers.py

```diff
@@ -58,6 +58,8 @@
 
 
 if TYPE_CHECKING:
+    from importlib_metadata import EntryPoint
```

Comment:
Here too, is it safe to comment something like:

```suggestion
    from importlib_metadata import EntryPoint
    # TODO: This can be switched to `importlib.metadata` from stdlib in Python 3.10+
```

?

Reply:
I think it's more like 3.11, but the doc is unclear about that.